react/features/dial-out/actionTypes.js

```diff
@@ -0,0 +1,48 @@
+import { Symbol } from '../base/react';
+
+/**
+ * The type of the action which signals a check for a dial-out phone number has
```

Comment:
I don't get this one: "Maybe these actions should specify dial-out number?" What do you mean?